docker-compose.yml

```diff
@@ -13,3 +13,11 @@ services:
       - UI_PORT=${UI_PORT}
       - WORK_DIR=${WORK_DIR}
       - SIGLENS_VERSION=${SIGLENS_VERSION}
+
+  otel-collector:
```

Comment:
we should name this as `siglens-otel-collector` and use that reference everywhere